wcvendors/wcvendors
# Fixed: CSS Issue in admin update bubbles

### All Submissions:


### Changes proposed in this Pull Request:



Closes #733 .

### How to test the changes in this Pull Request:

1. Load admin backend
2. See css issue is gone
